app/scripts/HiGlassComponent.js

```diff
@@ -3057,7 +3057,7 @@ class HiGlassComponent extends React.Component {
     }
 
     if (
-      !(+start1Abs >= 0 && +end1Abs >= 0)
+      isNaN(start1Abs) || isNaN(end1Abs)
     ) {
       const coords = [start1Abs, end1Abs].join(', ');
       console.warn([
```

Comment:
FWIW, I think the backtick syntax can span multiple lines.

Reply:
Oh, good to know!